### ISSUE TITLE
For Improvement

### ISSUE THREAD

**Issue by zhamri** [STIW2024/Agro-Kickstarter#5]
Please watch the [Presentation Video](https://github.com/STIW2024/Agro-Kickstarter/wiki/Youtube-Link) and test the [web application](https://github.com/STIW2024/Agro-Kickstarter/wiki/Website). Then give some good comments/ideas for the betterment of this project.

**Comment by JohnWick27** [comment]
Yeah. good afford. bro. keep it up.

**Comment by JohnWick27** [comment]
the problem. is everyone can be admin and user. and anyone can delete whole database without any permission. what do you think about that? 
  @zhenkit1996

**Comment by JohnWick27** [comment]
and maybe you can try to put simple session function just and an idea.

**Comment by ezzat787** [comment]
the layout of the web pages is very nice and also the website is ease to navigate... overall the website is good

**Comment by kzkit** [comment]
@JohnWick27 yeah i noticed that problem but I really didn't have time to play with the security part as I'm busy with my finals and doing this alone. Hopefully you have any references I can have to improve it? After my finals I plan to make the website nicer and better as a side project

**Comment by kzkit** [comment]
@ezzat787 thank you bro!

**Comment by JohnWick27** [comment]
yeah @zhenkit1996 of course later i will share some references.

**Comment by kzkit** [comment]
@JohnWick27 i've fixed the problem. help me check it out!
  
to go admin mode use 
username = kong
pw  = 123
role = admin

**Comment by jairam0905** [comment]
@zhenkit1996 Well done kit. It is a very simple and nice design with more functionality. Nice project and keep it up bro!

**Comment by kopilim96** [comment]
nice Project and idea about the project. What you have done in this project has differentiate you and others people in this class. What a nice Job!

**Comment by ChoyCheeWei** [comment]
well done kit. the design is simple, nice but full functionality. But I think the Get Started button should move it to the top right corner and the view project button can move to the top side(just my opinion). The overall project is good, keep it up!

**Comment by yaopeng96** [comment]
Simple and clear design. Visual structure and flow are clear. Nice project! Well done bro!

**Comment by nadiahnaz** [comment]
Nice website! I think of the homepage upper right corner beside login button should have register button for a user who didn't have an account for more register navigation beside access using get started button. But overall is awesome. Good job!

**Comment by AiniArasad** [comment]
Woahh! Nice website with suitable colour, agree with nadiah, supposed to have register button for user didn’t have an account. I like the simple design but full functionality. Good job kit!

**Comment by thiviya** [comment]
Good job Kit! It's a very nice and simple website plus it's easy to understand. I just have the same thought that you should put the register button next to login button so it will be easy for the users of the website. The overall project is good, keep it up!

**Comment by Nadzifa707** [comment]
Nice website of Agro Kickstarter !  It is simple and easy to navigate. Good job 👏👏

**Comment by JohnWick27** [comment]
@zhenkit1996  here is some line of code you may use for session function !
``` java
<%
       String username = (String)session.getAttribute("username");
if(username==null)
{
    response.sendRedirect("../index.jsp");
    return; //the return is important; forces redirect to go now
    
}
%>
```

**Comment by Koksheng** [comment]
That's really great. You guys doing fantastic job. Hope can learn something from you.

**Comment by afiqznw** [comment]
U did a great job Kit...your project looks awesome! Thumbs up!

**Comment by ernie2796** [comment]
Nice website! Yet, I think you may bring user to the homepage after the registration process. It may confused the user whether they already registered or not. Or you may give a message that they are already registered or not. Overall, you guys have done a great job!

**Comment by Nurzyra** [comment]
Nice website..clear and simple design.It easy to navigate your website.

**Comment by NgZhengYi** [comment]
Nice website, great function and a good layout design
